### PR TITLE
misc testing fixes

### PR DESCRIPTION
**Pull request by reidpr**

### CODE REVIEW

**Comment by reidpr** [comment]
I'd like reviews from (1) @mphinney1100 and (2) either of @heasterday or @j-ogas.

File: configure.ac [hunk 1]
```diff
@@ -210,7 +210,7 @@ AC_DEFUN([CH_CHECK_VERSION], [
 AC_PROG_CC
 
 # Set up CFLAGS.
-ch_cflags='-std=c99 -Wpedantic -Wall'
```
Comment (j-ogas) [inline configure.ac:213]: Why remove `-Wpendantic`? IIRC you were pretty against removing it in the past.

**Review by j-ogas — CHANGES_REQUESTED** [review]

File: configure.ac (hunk 1 above)
Comment (reidpr) [reply configure.ac:213]: Yeah I know. I wish I remembered why. 🙃

**Review by j-ogas — APPROVED** [review]
LGTM.

**Review by mphinney1100 — APPROVED** [review]

**Pull request merged**